src/styles/blocks/testimonials.scss

```diff
@@ -0,0 +1,240 @@
+.testimonials {
+  padding-bottom: 120px;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__subhead {
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 150%;
+    color: #253757;
+    margin-bottom: 48px;
+  }
+
+  &__card {
+    width: 100%;
+    background: white;
+    box-shadow: 0px 9px 18px rgba(37, 41, 49, 0.03);
+    border-radius: 16px;
+    text-align: center;
+    margin: 0 auto 24px;
+    z-index: 0;
+    transition: .3s;
+  }
+
+  &__card-container {
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: column;
+    align-items: center;
+    padding: 76px 42px 32px;
+  }
+
+  &__img {
+    width: 110px;
+    height: 110px;
+    border-radius: 50%;
+    object-fit: cover;
+    object-position: center;
+    margin-bottom: 100px;
+    position: relative;
+
+    &::before {
+      position: absolute;
+      content: url(../images/testimonials/testimonials-img-rectangle-mobile.svg);
+      right: 3.2em;
+      bottom: 1.8em;
+      z-index: -1;
+    }
+
+    &::after {
+      position: absolute;
+      content: url(../images/testimonials/testimonials-img-rectangle-mobile.svg);
+      left: 3.1em;
+      top: 2.3em;
+      z-index: -1;
+    }
+  }
+
+  &__img--1 {
+    background: url(../images/testimonials/testimonial-ava-1.jpg);
+    background-repeat: no-repeat;
```

Comment:
Please move this 3 rules to general selector for prevent repeating. It's for all img--n selectors